product_profile/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_product_brand_product_manager","product.profile","model_product_profile","base.group_user",1,0,0,0
```

Comment:
product_brand_product_manager ? 

missing a security rule to update/create profiles